api-definitions/db_positions/api.yaml

```diff
@@ -0,0 +1,292 @@
+    pattern: "^[0123][0-9]{17}$"
+consumes:
+  - application/json
+produces:
+  - application/json
+securityDefinitions:
+  ApiKey:
+    type: apiKey
+    name: X-Functions-Key
+    in: header
+    description: The API key to access this function app.
+definitions:
+  Timestamp:
+    $ref: "https://raw.githubusercontent.com/pagopa/io-functions-commons/v10.2.3/openapi/definitions.yaml#/Timestamp"
+  FiscalCode:
+    $ref: "https://raw.githubusercontent.com/pagopa/io-functions-commons/v10.2.3/openapi/definitions.yaml#/FiscalCode"
+  OrganizationFiscalCode:
+    $ref: 'https://raw.githubusercontent.com/pagopa/io-functions/v0.64.0/api/definitions.yaml#/OrganizationFiscalCode'
+  PaymentNoticeNumber:
+    $ref: "https://raw.githubusercontent.com/teamdigitale/io-functions-commons/master/openapi/definitions.yaml#/PaymentNoticeNumber"
+  ProblemJson:
+    $ref: "https://raw.githubusercontent.com/pagopa/io-functions-commons/v10.2.3/openapi/definitions.yaml#/ProblemJson"
+  PaginationResponse:
+    $ref: "https://raw.githubusercontent.com/pagopa/io-functions-commons/v10.2.3/openapi/definitions.yaml#/PaginationResponse"
+  PaymentAmount:
+    description: |-
+      Amount of payment in euro cent. PagoPA accepts up to 9999999999 euro cents.
+    type: integer
+    minimum: 1
+    maximum: 9999999999
+    example: 15000
+  Iban:
+    type: string
+    pattern: '[a-zA-Z]{2}[0-9]{2}[a-zA-Z0-9]{1,30}'
+    example: IT60X0542811101000000123456
+  DetailResponse:
+    type: object
+    properties:
+      detail:
+        type: string
+    required:
+      - detail
+  DepartmentName:
+    type: string
+    description: |-
+      The department inside the organization that runs the service. Will
+      be added to the content of sent messages.
+    minLength: 1
+    example: ufficio tributi comune di Roma
+  Taxonomy:
+    type: string
+    pattern: "^[0-9]{3}-[a-zA-Z]$"
+    example: 010-TARI/TEFA
+    description: Payment Tax see [Tabella - Tassonomia dei servizi di incasso](https://docs.google.com/spreadsheets/d/13xOd__Qd4pwKHr3wjE-73NAB2O7UKmIt/edit#gid=1444196601)
+  Transfer:
+    type: object
+    properties:
+      partial_amount:
+        $ref: "#/definitions/PaymentAmount"
+      transfer_IBAN:
+        $ref: "#/definitions/Iban"
+      pa_fiscal_code:
+        $ref: "#/definitions/OrganizationFiscalCode"
+      reason:
+          type: string
+          minLength: 1
+          example: Causale pagamento TARI comune di ROMA 
+      taxonomy:
+          $ref: "#/definitions/Taxonomy"
+    required:
+      - partial_amount
+      - transfer_IBAN
+      - fiscal_code
+      - taxonomy
+  Debtor:
+    type: object
+    properties:
+      name:
+        type: string
+        minLength: 1
+        example: Mario
+      surname:
+        type: string
+        minLength: 1
+        example: Rossi
+      fiscal_code:
+        $ref: "#/definitions/FiscalCode"
+      type:
+        type: string
+        enum:
+          - FISICA
+          - GIURIDICA
```

Comment:
is it mandatory to put these in italian?

Reply:
no

Reply:
can we change them to english for consinstency?

Reply:
@pasqualespica  in other projects we use PERSON_NATURAL and PERSON_LEGAL.